source/client/controllers/signup-controller.js

```diff
@@ -0,0 +1,53 @@
+Space.messaging.Controller.extend(Space.accountsUi, 'SignupController', {
```

Comment:
No that's fine here since it also sends commands to the server! That's really the classic example of a controller :wink: